helm/cmp-platform/templates/NOTES.txt

```diff
@@ -20,7 +20,7 @@ kubectl get ing
 
 When the pods are ready, you can open up the consoles:
 
-http://openam.{{.Release.Namespace}}{{.Values.openam.domain}}/openam
+http://login.{{.Release.Namespace}}{{.Values.openam.domain}}/
```

Comment:
Want to put https for these? That's what the Quick Start's going to use (I have users deploy the cert-manager).